quick_tour/interacting_with_the_index.ipynb

```diff
@@ -21,11 +21,11 @@
     "and it lets you query their nearest neighbors.\n",
```

Comment:
<p>Shouldn't it be ids=["A"] instead of ids=["A", ]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>